src/prefect/artifacts.py

```diff
@@ -0,0 +1,160 @@
+from typing import Union
+
+from prefect import context, Client
```

Comment:
Isn't it dangerous to import the context this way?

Reply:
Hmm doesn't seem to affect my runs at all whether it's imported here or inside the functions that use it

Reply:
I remember seeing that it would always be fully imported / it changes when the context is loaded? Perhaps it only really matters within Tasks